src/lib.rs

```diff
@@ -157,27 +157,6 @@ pub type ArrayN<T, const N: usize> = Array<T, <[T; N] as AssocArraySize>::Size>;
 /// let arr: Array<u8, U3> = Array([1, 2, 3]);
 /// ```
 ///
-/// ## [`AsRef`] impls
-///
-/// The [`AsRef`] trait can be used to convert from `&Array<T, U>` to `&[T; N]` and vice versa:
-///
-/// ```
-/// use hybrid_array::{Array, ArraySize, AssocArraySize, ArrayN, sizes::U3};
-///
-/// pub fn get_third_item_hybrid_array<T, U: ArraySize>(arr_ref: &Array<T, U>) -> &T {
-///     &arr_ref[2]
-/// }
-///
-/// pub fn get_third_item_const_generic<T, const N: usize>(arr_ref: &[T; N]) -> &T
-/// where
-///     [T; N]: AssocArraySize + AsRef<ArrayN<T, N>>
-/// {
-///     get_third_item_hybrid_array(arr_ref.as_ref())
-/// }
-///
-/// assert_eq!(get_third_item_const_generic(&[1u8, 2, 3, 4]), &3);
-/// ```
-///
```

Comment:
I guess I can add something back here, but the impls in the other direction are still valid

Reply:
Ugh, played with this myself and ended up removing it.